addons/sale/views/sale_portal_templates.xml

```diff
@@ -138,7 +138,23 @@
                     <t t-set="classes" t-value="'col-lg-4 col-xxl-3 d-print-none'"/>
 
                     <t t-set="title">
-                        <h2 t-field="sale_order.amount_total" data-id="total_amount" class="mb-0 text-break"/>
+                        <t t-set="amount_len" t-value="len(sale_order.currency_id.symbol + ' ' + '{:.2f}'.format(sale_order.amount_total))"/>
+                        <t t-if="amount_len > 23">
+                            <t t-set="font_class" t-value="'h6'"/>
+                        </t>
+                        <t t-elif="amount_len > 19">
+                            <t t-set="font_class" t-value="'h5'"/>
+                        </t>
+                        <t t-elif="amount_len > 15">
+                            <t t-set="font_class" t-value="'h4'"/>
+                        </t>
+                        <t t-elif="amount_len > 11">
+                            <t t-set="font_class" t-value="'h3'"/>
+                        </t>
+                        <t t-else="">
+                            <t t-set="font_class" t-value="'h2'"/>
+                        </t>
```

Comment:
This solution is clearly too magic & complex for the current issue.
Furthermore, this has to stay coherent between sale, purchase & account portals, so I requested a review from CHGO as he's the one behind the latest design fixes on this part of the portal templates.

Reply:
Thanks for this fix @pash-odoo. As @Feyensv mentioned, this seems a bit too complex for this "issue". Are there tickets related to this ? I see in the task description that the feedback comes from Odoo.com but the `h2` there is way bigger than the one we use in the front-end so the fix shouldn't be done here but eventually on the Odoo.com website.

The current behaviour looks like the expected one IMO, changing the font-size directly will result in a weird visual result when navigating through invoice, which I'd prefer to avoid.

Anyway, I'd keep things simple here, reducing the heading tag to `h3` or `h4` should do the job if this is really an issue. Better have the amount wrapping on a second line rather than truncate it.

Reply:
Sorry, I probably created this task too quickly without noticing the problem was maybe specific to Odoo.com. 
We can cancel it